bundles/org.openhab.binding.nikobus/src/main/java/org/openhab/binding/nikobus/internal/handler/NikobusPushButtonHandler.java

```diff
@@ -177,13 +263,41 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
     }
 
+    // called when there is a message on the Nikobus
     private void commandReceived() {
-        if (thing.getStatus() != ThingStatus.ONLINE) {
```

Comment:
Button is put online when first Nikobus command is received - meaning the button is mapped correctly. AFAIK there is no other way to check if configured address is correct.